src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -135,6 +140,20 @@ subroutine init_atm_static(mesh, dims, configs)
 
  real (kind=RKIND) :: xPixel, yPixel, zPixel
 
+ type (mpas_kd_type), dimension(:), pointer :: kd_points
+ type (mpas_kd_type), pointer :: tree
+ type (mpas_kd_type), pointer :: res
+
+ type (mpas_geotile_mgr_type) :: mgr
+ type (mpas_geotile_type), pointer :: tile
+
+ real (kind=RKIND) :: tval
+ integer, pointer :: tile_bdr
+ integer, pointer :: tile_nx, tile_ny
+
+ logical :: all_pixels_mapped_to_halo_cells = .true.
```

Comment:
It probably doesn't matter, but just as a matter of convention, it might be good to remove the initialization of this variable so that it doesn't take on the `SAVE` attribute implicitly.